docs/examples.md

```diff
@@ -182,6 +182,27 @@ new PipelineJobBuilder(
 
```

Comment:
meta: please make the PR title more descriptive; please squash commits

nits: please fix formatting inconsistencies in the builder class's `build` method